<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Smart Card reader tester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="textReceived">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>760</width>
      <height>300</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textSended">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>760</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>760</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_received">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>660</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Received:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sended">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>760</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sended:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Open port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portSpeed">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="portName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Close port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textToSend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>660</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_to_send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>760</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data to sent (hex format, space delimited, without 0x; example: 'BA 02 01 B9'):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearReceived">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonOpen</tabstop>
  <tabstop>buttonSend</tabstop>
  <tabstop>textSended</tabstop>
  <tabstop>textReceived</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
